src/phoenix/server/api/types/PromptVersion.py

```diff
@@ -0,0 +1,35 @@
+# Part of the Phoenix PromptHub feature set
+
+from enum import Enum
+from typing import Optional
+
+import strawberry
+from strawberry.relay import Node, NodeID
+from strawberry.scalars import JSON
+
+
+@strawberry.enum
+class PromptTemplateType(str, Enum):
+    TEXT = "text"
+    JSON = "json"
```

Comment:
is this supposed to match what's in the DB?
```
        sa.Column(
            "template_type",
            sa.String,
            sa.CheckConstraint(
                "template_type IN ('chat', 'str')",
                name="template_type",
            ),
            nullable=False,
        ),
```

Reply:
added